### ISSUE TITLE
Classification of a sequence of graphs

### ISSUE THREAD

**Issue by Borzyszkowski** [benedekrozemberczki/pytorch_geometric_temporal#178]
Hello!

In my project, I work on motion classification and was wondering if _pytorch geometric temporal_ could be used for this application.
Namely, my dataset consists of a sequence of graphs such that each graph represents a hand pose, where joints are given as nodes and bones are given as edges. The graph is static with temporal signal (3 dynamic features in each of 21 nodes).

I implemented a GCN for graph classification using _pytorch geometric_, which takes the following data representation:
`Data(x=[21, 3], edge_index=[2, 40], y=[1])`

Now, I would like to use the temporal relationship between the graphs and classify their whole sequence instead of a single graph.
However, I encountered the following issues:
-`StaticGraphTemporalSignal` requires to define targets (labels) for each graph, which is in conflict with my case
-I could not pass entire sequence of graphs to the neural network, eg. DCRNN. For instance, examples given on the [website](https://pytorch-geometric-temporal.readthedocs.io/en/latest/notes/introduction.html#epidemiological-forecasting) calculate loss for each graph separately, and not for the sequence of graphs as a whole. 

My approach is quite similar to the work presented in [this paper](https://arxiv.org/abs/1801.07455) and I was wondering if using _pytorch geometric temporal_ for this case would be possible. Any advice will be appreciated!

Best,
Bartek

**Comment by Borzyszkowski** [comment]
I managed to run [ST-GCN](https://pytorch-geometric-temporal.readthedocs.io/en/latest/_modules/torch_geometric_temporal/nn/attention/stgcn.html#STConv) and [2s-AGCN](https://pytorch-geometric-temporal.readthedocs.io/en/latest/_modules/torch_geometric_temporal/nn/attention/tsagcn.html#AAGCN), according to the papers [[1](https://arxiv.org/pdf/1801.07455.pdf) and [2](https://arxiv.org/pdf/1805.07694.pdf)] respectively.

I used the following graph:
`Data(x=[4000, 21, 3], edge_index=[4000, 2, 40], y=[4000, 1])`
where in my case 4000 is a fixed size of a sequence.

**Comment by gbastiandillo** [comment]
Hi @Borzyszkowski , how're you? 
*Can you share how many group of sequences did u used? (explained in other way, if each sequence it's a video, how many videos did u used?)
*Do you think can share your final model architecture? 

Thanks!
